### PR TITLE
APPLICATION: Created Exeptions

### LINKED ISSUE #1: DOMAIN: Entities, Constans, Enums

### LINKED ISSUE #3: DATAACCESS: Created Interfaces, Utils

### LINKED ISSUE #5: APPLICATION: Create All Exceptions.

### PR DESCRIPTION
**Pull request by DotnetUzDev**
Closed #1 
Closed #3 
Closed #5

### CODE REVIEW

File: src/HeavyService.Application/Exeptions/AllReadyExeption.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Net;
+
+namespace HeavyService.Application.Exeptions;
+
+public class AllReadyExistsExeption : ClientExeption
+{
+    public override HttpStatusCode StatusCode { get; } = HttpStatusCode.NotFound;
+
```
Comment (ShohruhUzDev) [inline src/HeavyService.Application/Exeptions/AllReadyExeption.cs:8]: delete empty line

File: src/HeavyService.Application/Exeptions/BadRequestExeption.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Net;
+
+namespace HeavyService.Application.Exeptions;
+
+public class BadRequestExeption : ClientExeption
+{
+    public override HttpStatusCode StatusCode { get; } = HttpStatusCode.BadRequest;
+
```
Comment (ShohruhUzDev) [inline src/HeavyService.Application/Exeptions/BadRequestExeption.cs:8]: bu qator uchirilsin

File: src/HeavyService.Application/Exeptions/ClientExeption.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Net;
+
+namespace HeavyService.Application.Exeptions;
+
+public abstract class ClientExeption : Exception
+{
+    public abstract HttpStatusCode StatusCode { get; }
+
+    public abstract string TitleMessage { get; protected set; }
+
```
Comment (ShohruhUzDev) [inline src/HeavyService.Application/Exeptions/ClientExeption.cs:10]: bu ham

File: src/HeavyService.Application/Exeptions/ExpiredExeption.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Net;
+
+namespace HeavyService.Application.Exeptions;
+
+public class ExpiredExeption : ClientExeption
+{
+    public override HttpStatusCode StatusCode { get; } = HttpStatusCode.Gone;
+
```
Comment (ShohruhUzDev) [inline src/HeavyService.Application/Exeptions/ExpiredExeption.cs:8]: buni ham uchiring

File: src/HeavyService.Application/Exeptions/NotFoundExeption.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Net;
+
+namespace HeavyService.Application.Exeptions;
+
+public class NotFoundExeption : Exception
+{
+    public HttpStatusCode StatusCode { get; } = HttpStatusCode.NotFound;
+
```
Comment (ShohruhUzDev) [inline src/HeavyService.Application/Exeptions/NotFoundExeption.cs:8]: uchiring

**Review by ShohruhUzDev — CHANGES_REQUESTED** [review]

**Review by DotnetUzDev — COMMENTED** [review]
hamma bo'sh qatorlarni tozaladim

**Pull request merged**